### ISSUE TITLE
FlowInterleaveSpec

### ISSUE THREAD

**Issue by drewhk** [akka/akka#19531]
https://jenkins.akka.io:8498/job/akka-multi-node/1267/consoleFull

```
[info] - must work with one immediately failed and one nonempty publisher *** FAILED *** (28 milliseconds)
[info]   java.lang.AssertionError: assertion failed: expected OnError(akka.stream.testkit.BaseTwoStreamsSetup$$anon$1: test
[info] ), found OnNext(1)
[info]   at scala.Predef$.assert(Predef.scala:165)
[info]   at akka.testkit.TestKitBase$class.expectMsg_internal(TestKit.scala:351)
[info]   at akka.testkit.TestKitBase$class.expectMsg(TestKit.scala:327)
[info]   at akka.testkit.TestKit.expectMsg(TestKit.scala:735)
[info]   at akka.stream.testkit.TestSubscriber$ManualProbe.expectError(StreamTestKit.scala:361)
[info]   at akka.stream.scaladsl.FlowInterleaveSpec$$anonfun$1$$anonfun$apply$mcV$sp$9.apply$mcV$sp(FlowInterleaveSpec.scala:115)
[info]   at akka.stream.scaladsl.FlowInterleaveSpec$$anonfun$1$$anonfun$apply$mcV$sp$9.apply(FlowInterleaveSpec.scala:106)
[info]   at akka.stream.scaladsl.FlowInterleaveSpec$$anonfun$1$$anonfun$apply$mcV$sp$9.apply(FlowInterleaveSpec.scala:106)
```

**Comment by drewhk** [comment]
similar to https://github.com/akka/akka/issues/19530

**Comment by rkuhn** [comment]
This is already fixed in #19494 which I am desperately trying to get merged.

**Comment by rkuhn** [comment]
or no, actually that was #19518
